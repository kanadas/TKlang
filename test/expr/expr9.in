(\f x -> f (f x)) (\y -> y) 7
