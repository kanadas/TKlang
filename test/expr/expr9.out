expr/expr9.in
7
